testsuite/tests/cxx-api/all-includes.h.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if command -v cygpath >/dev/null 2>&1; then
+  ocamlsrcdir="$(cygpath -u "${ocamlsrcdir:?}")"
+  test_build_directory="$(cygpath -u "${test_build_directory:?}")"
+fi
+
+exec find "${ocamlsrcdir:?}"/runtime/caml "$ocamlsrcdir"/otherlibs/*/caml \
```

Comment:
`find` is a terrifyingly difficult tool to get portably right (`-execdir` is a GNUism, I think?) - I'm tempted rather to generate this header as part of the build, where you could just `$(wildcard` in the build system?

Reply:
Good idea, and it happened that my `PATH` was corrupted, which `find` likes to complain about.